### ISSUE TITLE
no buildable Go source files

### ISSUE THREAD

**Issue by oren** [nii236/nii-finance#3]
I follow the readme and get:

```
go get -d github.com/nii236/nii-finance
package github.com/nii236/nii-finance: no buildable Go source files in /home/oren/p/go/src/github.com/nii236/nii-finance
```

**Comment by oren** [comment]
also when i fork it and run `go build` i get the same error. i assume i first need to `cd` into some folder but not sure.

**Comment by nii236** [comment]
Thanks for the comment!

The whole project runs off `docker-compose`, and there are no buildable go source files in the root folder. The buildable folders are in each of folders in `./service`.

Try and remove the original `nii-finance` folder first, then `go get -d` again. The `-d` flag just downloads so it shouldn't complain. I just tried it myself and I got the same error until I removed the `nii-finance` folder first.
